assets/js/components/settings/settings-admin.js

```diff
@@ -190,10 +190,7 @@ class SettingsAdmin extends Component {
 												googlesitekit-settings-module__meta-item
 												googlesitekit-settings-module__meta-item--nomargin
 											">
-												<Optin
-													id="opt-in"
-													name="optin"
```

Comment:
Note that this name differs from the name used as default (and thus used now that you removed this). That should be fine, but wanted to flag it.

Reply:
As @aaemnnosttv mentioned: the names weren't used for anything, so I wanted to remove them so they didn't appear to be needed by the next developer wandering into here 😄